lrlex/src/lib/lexer.rs

```diff
@@ -281,13 +284,37 @@ impl<
                         }
                     }
                 }
-                if let Some(target_state_id) = r.target_state {
-                    state = match self.get_start_state_by_id(target_state_id) {
+                if let Some((target_state_id, op)) = &r.target_state {
+                    // TODO Handle state stack operations
```

Comment:
Still a TODO, seems like it is?

Reply:
Resolved in f3794c8343d0c1ee1980237527b5fd8218290ad1